chemprop/uncertainty/predictor.py

```diff
@@ -9,126 +9,133 @@
 from chemprop.utils.registry import ClassRegistry
 
 
-class UncertaintyPredictor:
-    def __call__(self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer):
-        return self._calc_prediction_uncertainty(dataloader, models, trainer)
+class UncertaintyPredictor(ABC):
+    """
+    A class for making model predictions and associated uncertainty predictions.
+    """
```

Comment:
I'm not sure on what the correct docstring for this class should be. I would like to not overload the term "predictor" inside the codebase.

```suggestion
    """A helper class for making model predictions and associated uncertainty predictions."""
```

Reply:
This sounds like a good idea to not overload the term predictor. How do you feel about the term "estimator". We would make this change right before merging this into main though, given there are many open PRs that depend on this, so it is probably easier to make the change at the end all at once instead of needing to update each PR. 

`chemprop.uncertainty.predictor -> chemprop.uncertainty.estimator`
`UncertaintyPredictor -> UncertaintyEstimator`

There is also the question of whether `calibrator.py`, `evaluator.py` and `estimator.py` should be made plural -> `calibrators.py`, `evaluators.py`, `estimators.py` to match the plural names in `chemprop.nn`.